### ISSUE TITLE
Does this warrant being a standalone module?

### ISSUE THREAD

**Issue by rufuspollock** [okfn/datapackage-outfile-js#1]
@pwalsh I'm wondering if this is just a gist or a pattern in a tutorial as opposed to a full module ...

**Comment by pwalsh** [comment]
No, it does not. Whatever here is worth keeping I'd merge it into https://github.com/frictionlessdata/datapackage-js as a `save` method or similar.
